staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -181,6 +182,21 @@ func (c *RESTClient) Patch(pt types.PatchType) *Request {
 	return c.Verb("PATCH").SetHeader("Content-Type", string(pt))
 }
 
+// Apply begins a PATCH request with the apply patch type.
+func (c *RESTClient) Apply(force bool, fieldManager string) *Request {
+	req := c.Patch(types.ApplyPatchType)
+
+	if force {
+		req.Param("force", "true")
+	}
+
+	if manager != "" {
```

Comment:
this really shouldn't happen, the request will fail if the manager isn't specified.

Reply:
so should I blindly pass the value of `fieldManager` or should I log an error and return a `nil` request?

Reply:
I would pass blindly